alicloud/resource_alicloud_slb_listener.go

```diff
@@ -256,7 +256,7 @@ func resourceAliyunSlbListener() *schema.Resource {
 						"retrive_slb_proto": {
 							Type:     schema.TypeBool,
 							Optional: true,
-							Default:  false,
+							Default:  true,
```

Comment:
I check the slb_listener docs, retrive_slb_proto will be set to false by default. it's XForwardedFor_proto

Reply:
Hmm, that's weird, can you point me which docs you are referring to?

Reply:
it's CreateLoadBalancerHTTPListener, the retrive_slb_proto is correspond XForwardedFor_proto

Reply:
Can you show me the direct link of the code or docs? 

Because I can see here https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/website/docs/r/slb_listener.html.markdown#block-x_forwarded_for

<img width="933" alt="屏幕快照 2019-06-12 上午10 20 51" src="https://user-images.githubusercontent.com/3715820/59319060-ee0cb180-8cfb-11e9-93f7-99711f1e7f4a.png">

It's `true` by default.

Reply:
thank you,This is my mistake,I will update the docs